external/netfx/netfx.depproj

```diff
@@ -25,5 +25,11 @@
     <FileToExclude Include="System.EnterpriseServices.Thunk" />
     <FileToExclude Include="System.EnterpriseServices.Wrapper" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">
```

Comment:
The problem is that we need this targeting pack in order to build the desktop and netstandard shims, which are required in order to be able to build the uap tree. We should probably consider renaming this project in order to avoid confusion.